lib/src/excel.dart

```diff
@@ -126,6 +137,32 @@ class ExcelDart {
     }));
   }
 
+  /// Highlight rows that are underpacing.
+  ///
+  /// Structured reference (i.e. Table[@Column]) is not allowed in Excel
+  /// conditional formatting. Regular reference (i.e. $A5) has to be used
+  /// instead. As table expands, Excel would automatically extends the
+  /// conditional formatting rule to cover the entire range.
+  static void _highlightUnderpacingRows(
+      String tableName, String threshold) async {
+    await ExcelJS.run(allowInterop((context) async {
+      final table = context.workbook.tables.getItem(tableName);
+      final range = table.getDataBodyRange();
+      final format = range.conditionalFormats.add('Custom');
+
+      format.custom.rule.formula = '''
+      ($_spentColumn/$_budgetColumn) *
+      (DATEDIF($_startDateColumn, $_endDateColumn, "D") / 
```

Comment:
what kind of division is this? is it integer division? 
also what happens when the difference is 0?
(does the "D" mean double?) Since this is not a common library may be helpful to explain non-obvious args in a comment so they arent treated as "magic" inputs.